clients/js/src/AdminClient.ts

```diff
@@ -64,7 +65,7 @@ export class AdminClient {
       basePath: path,
     });
 
-    this.api = new DefaultApi(apiConfig);
+    this.api = new DefaultApi(apiConfig, undefined, chromaFetch);
```

Comment:
for my understanding, why does this result in well formatted errors?

Reply:
`chromaFetch` is a wrapper around `fetch()` that will throw Errors based on the response status code

by default, the autogenerated client seems to throw Responses

Reply:
got it